documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts/_index.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Common Mounts"
+date = 2019-02-23T16:45:16-05:00
+weight = 70
+pre = "<b> </b>"
++++
+
+Use common mounts to automatically include the directory content from additional images. This is a useful alternative for including Model in Image model files, or other types of files, in a Pod without requiring modifications to the Pod's base image specified using `domain.spec.image`. This feature internally uses a Kubernetes [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volume and Kubernetes [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) to share the files from the additional images with the Pod.
+
+When using the common mounts feature with Model In Image use case, the WebLogic image would be the Domain resource's `domain.spec.image` field. The model files, application archives, and the WDT installation files can be provided in a separate image specified as part of the common mounts. The application archive could also be provided by a second container image in the common mounts section. Specific advantages of the common mounts feature for Model In Image domains are:
+  - Use a WebLogic install image, or patch a WebLogic install image, without needing to include model artifacts within the image.
+  - Share one large WebLogic install image with multiple different model configurations.
+  - Distribute model files, application archives, and WebLogic Deploy Tooling executable using very small images instead of a large image that also contains a WebLogic install.
+
+### Configuration
+Here's an example configuration for the Common Mounts. 
+
+```
+  serverPod:
+    commonMounts:
+    - image: model-in-image-files:1
+      imagePullPolicy: IfNotPresent
+      command: cp -R $COMMON_MOUNT_PATH/* $COMMON_MOUNT_TARGET_PATH
+      volume: commonMountsVolume1
+```
+
+The `volume` field in the common mounts refers to the name of a common mount volume defined in `spec.commonMountVolumes` section. Here's an example configuration for the common mount volumes.
+```
+  spec:
+    commonMountVolumes:
+    - name: commonMountsVolume1
+      mountPath: /common
+      medium: Memory
```

Comment:
Should not use 'Memory' - this is not the default, and pod memory may be at a premium